### ISSUE TITLE
Can anyone get undetected-chromedriver to work on the website fmovies.to?

### ISSUE THREAD

**Issue by ariyasas** [ultrafunkamsterdam/undetected-chromedriver#603]
So far, I've found that uc works on pretty much every website _except_ for a site called fmovies.to. The site seems to always detect chromedriver and will redirect me back to the homepage. I know it's not the extensions messings things up since chromedriver gets blocked even without them.  Is there an error on my end? Or is this just a website that uc can't get past?

    import undetected_chromedriver.v2 as uc

    options = uc.ChromeOptions()
    options.add_argument('--load-extension=/Users/User1/Desktop/ublock')

    driver = uc.Chrome(use_subprocess=True, options=options)

    driver.get('https://fmovies.to/movie/spider-man-homecoming-520k0/1-full')

**Comment by QIN2DIM** [comment]
This has nothing to do with uc. This site requires you to download a specific browser plug-in to access it properly.

![image](https://user-images.githubusercontent.com/62018067/164603656-b00b04cf-20bd-4aed-af7d-e404dba745b7.png)

**Comment by ariyasas** [comment]
No, you can watch movies perfectly fine without downloading any plugin if you're manually browsing the site through Chrome or a similar browser. The site's redirects only kick in if you try browsing using automation e.g. through chromedriver or uc-chromedriver.

**Comment by sebdelsol** [comment]
I confirm what @QIN2DIM found: you need a (dubious) program to be installed for it works, after that you don't seem to be detected. Btw you shouldn't use `use_subprocess=True`, it'll make your dirver more detectable.
